README.md

```diff
@@ -26,7 +26,7 @@
 
 | Date      |     Doc Revision      | Version |   Details |
 |----------|:-------------:|------:|:------|
-| November 2023 | 011 | 23.11 | - Support DC NS (NoSession) APIs.  <br> - Support  DC compressBound APIs. <br> - Support Symmetric Crypto SM3 & SM4. <br> - Support Asymmetric Crypto SM2. <br> - Bug Fixes. See [Resolved Issues](#resolved-issues). |
+| November 2023 | 011 | 23.11 | - Support DC NS (NoSession) APIs.  <br> - Support  DC compressBound APIs. <br> - Support Symmetric Crypto SM3 & SM4. <br> - Support Asymmetric Crypto SM2. <br> - The ABI version changed in this release, so the shared object version (liqat.so) is incremented in this release. <br> - Bug Fixes. See [Resolved Issues](#resolved-issues). |
```

Comment:
HI John.
that should be on the 23.08 row.
Also typo -  it's libqat.so.
And you can leave out the second "in this release".